homeassistant/components/overkiz/manifest.json

```diff
@@ -11,13 +11,17 @@
   ],
   "documentation": "https://www.home-assistant.io/integrations/overkiz",
   "integration_type": "hub",
-  "iot_class": "cloud_polling",
+  "iot_class": "local_polling",
```

Comment:
This integration will actually have to support both local or cloud polling.  
In some case, Overkiz/Somfy integration doesn't work completly on local_polling
@frenck would it be possible and useful to add a new iot_class "local_or_cloud_polling" that will be used here? Or local_polling implies that there might be some cloud_polling too?
I think this is not a blocking point for now for this PR.

Reply:
This was indeed an open discussion point still. Should we set the `iot_class` to the best possible option, or to the one that most users will face?

And if we talk about most users, I would say that 80% is using Somfy, so that at least 60% (not users of TaHoma v1 or Connectivity Kit) can use the local integration.

Reply:
Curious about this as well: I recently contributed a "`local_push`" connection mechanism to an originally `local_polling` integration (currently still out of tree), and wanted to have this be an array or comma-separated string as well to be able to set multiple classes (rather than coming up with various new `io_class`es that are simply a combination of others?).